packages/node/test/unit/dev.test.ts

```diff
@@ -43,21 +43,250 @@ function testForkDevServer(entrypoint: string) {
+        }).toEqual({
+          status: 200,
+          'x-web-handler': 'Web handler using HEAD',
+        });
+      }
+      {
+        const response = await fetch(
+          `http://${address}:${port}/api/web-handlers-node`,
+          { method: 'OPTIONS' }
+        );
+        expect({
+          status: response.status,
+          body: await response.text(),
+          transferEncoding: response.headers.get('transfer-encoding'),
+          'x-web-handler': response.headers.get('x-web-handler'),
+        }).toEqual({
+          status: 200,
+          body: 'Web handler using OPTIONS',
+          transferEncoding: 'chunked',
+          'x-web-handler': 'Web handler using OPTIONS',
+        });
+      }
+    } finally {
+      child.kill(9);
+    }
+  }
+);
+
+(NODE_MAJOR < 18 ? test.skip : test)(
+  'web handlers for edge runtime',
+  async () => {
+    const child = testForkDevServer('./web-handlers-edge.js');
+    try {
+      const result = await readMessage(child);
+      if (result.state !== 'message') {
+        throw new Error('Exited. error: ' + JSON.stringify(result.value));
       }
 
+      const { address, port } = result.value;
+
+      {
+        const response = await fetch(
+          `http://${address}:${port}/api/web-handlers-edge`,
+          { method: 'GET' }
+        );
+        expect({
+          status: response.status,
+          body: await response.text(),
+          transferEncoding: response.headers.get('transfer-encoding'),
+          'x-web-handler': response.headers.get('x-web-handler'),
+        }).toEqual({
+          status: 200,
+          body: 'Web handler using GET',
+          transferEncoding: 'chunked',
+          'x-web-handler': 'Web handler using GET',
+        });
+      }
+      {
+        const response = await fetch(
+          `http://${address}:${port}/api/web-handlers-edge`,
+          { method: 'POST' }
+        );
+        expect({
+          status: response.status,
+          body: await response.text(),
+          transferEncoding: response.headers.get('transfer-encoding'),
+          'x-web-handler': response.headers.get('x-web-handler'),
+        }).toEqual({
+          status: 200,
+          body: 'Web handler using POST',
+          transferEncoding: 'chunked',
+          'x-web-handler': 'Web handler using POST',
+        });
+      }
+      // {
+      //   const response = await fetch(
+      //     `http://${address}:${port}/api/web-handlers-edge`, { method: 'DELETE' }
+      //   );
+      //   expect({
+      //     status: response.status,
+      //     body: await response.text(),
+      //     transferEncoding: response.headers.get('transfer-encoding'),
+      //     'x-web-handler': response.headers.get('x-web-handler')
+      //   }).toEqual({
+      //     status: 200,
+      //     body: 'Web handler using DELETE',
+      //     transferEncoding: 'chunked',
+      //     'x-web-handler': "Web handler using DELETE",
+      //   });
+      // }
```

Comment:
**suggestion (soft-blocking):** uncomment?

Are these `OPTIONS` calls not working?